zos_subsystems/ims/ims_provisioning-wazi/README.md

```diff
@@ -0,0 +1,131 @@
+- [**ims_apf**](roles/ims_apf/README.md) - adds authorization of IMS datasets to zOS
+
+- [**ims_catalog**](roles/ims_catalog/README.md) - allocates, loads, and deletes IMS catalog
+- [**ims_common**](roles/ims_common/README.md) - provides services such as start/stop IMS control regions and IMS Connection
+- [**ims_common_queue**](roles/ims_common_queue/README.md) - provides dataset definition and query or CQS
+- [**ims_dataset**](roles/ims_dataset/README.md) - creates and deletes IMS definition data sets and libraries
+- [**ims_dbrc**](roles/ims_dbrc/README.md) - sets up DBRC defaults and prepares DBRC
+- [**ims_exit**](roles/ims_exit/README.md) - prepares security and connection exit for IMS
+- [**ims_gen**](roles/ims_gen/README.md) - prepares DBDGEN, PSBGEN, and ACBGEN
+- [**ims_iefjobs**](roles/ims_iefjobs/README.md) - creates IEF jobs
+- [**ims_initialize**](roles/ims_initialize/README.md) - reserves ICON ports and sends PROCLIB templates to zOS
+- [**ims_online_change**](roles/ims_online_change/README.md) - enables the online change utility
+- [**ims_operations_manager**](roles/ims_operations_manager/README.md) - defines, starts, stops, and queries operations manager
+- [**ims_proclib**](roles/ims_proclib/README.md) - defines BPE configuration and copies PROCLIB
+- [**ims_racf**](roles/ims_racf/README.md) - prepares RACF security for IMS
+- [**ims_region**](roles/ims_region/README.md) - starts IMS regions
+- [**ims_resource_manager**](roles/ims_resource_manager/README.md) - defines, starts, stops and queries RM
+- [**ims_structured_call_interface**](roles/ims_structured_call_interface/README.md) - defines, starts, stops, and queries SCI
+- [**ims_sysdef**](roles/ims_sysdef/README.md) - prepares system definition data set
+- [**install-bzip2**](roles/install-bzip2/README.md) - copies and installs utility zip file on zOS
+- [**monitor_ims**](roles/monitor_ims/README.md) - checks to see if IMS is up or down.
+- [**provision_ims**](roles/provision_ims/README.md) - this is the main role to provision IMS.  It utilizes other roles to allocate datasets, prepare datasets and start all IMS services
+- [**save-templates-to-datasets**](roles/save-templates-to-datasets/README.md) - copies templates from USS to zOS data sets
+- [**send-template**](roles/send-template/README.md) - sends a template file from the local host to zOS
+- [**send-templates**](roles/send-templates/README.md)  - sends multiple templates in a directory from local host to zOS
+- [**stop_jmp**](roles/stop_jmp/README.md) - stop JMP (Java) region
+- [**stop_mpp**](roles/stop_mmp/README.md) - stop MPP region
+- [**submit-rexx**](roles/submit-rexx/README.md) - runs a REXX script on zOS
+
+
+Inside each role:
+
+* `./tasks` contains tasks that can be performed by the role.
+  Sub-folder names describe the use of the contained files.
+* `./tasks/main.yml` contains the default tasks performed by the role.
+
+
+## Ansible Collection Requirements
+* IBM z/OS core collection 1.4.0
```

Comment:
should we add 'beta.1' for these collections

Reply:
IMS Provisioning works fine with Core collection 1.4.0

Reply:
there's no such thing as ibm_zos_core 1.4.0 as see here:
```
ansible-galaxy collection install -U ibm.ibm_zos_core:1.4.0                                       
Starting galaxy collection install process
Process install dependency map
ERROR! Failed to resolve the requested dependencies map. Could not satisfy the following requirements:
* ibm.ibm_zos_core:1.4.0 (direct request)
(venv) (base) ➜  ims_provisioning-wazi git:(ims-provisioning-wazi) ✗ ansible-galaxy collection install -U ibm.ibm_zos_core:1.4.0-beta.1
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Downloading https://galaxy.ansible.com/download/ibm-ibm_zos_core-1.4.0-beta.1.tar.gz to /Users/adn/.ansible/tmp/ansible-local-46955az78w8y6/tmp6njhttp9/ibm-ibm_zos_core-1.4.0-beta.1-4nph_44a
Installing 'ibm.ibm_zos_core:1.4.0-beta.1' to '/Users/adn/.ansible/collections/ansible_collections/ibm/ibm_zos_core'
ibm.ibm_zos_core:1.4.0-beta.1 was installed successfully
```